javascript/builtins/PluralRules.json

```
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "OverconstrainedError": {
```

Copy and paste miss?

```suggestion
    "PluralRules": {
```